src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -12,21 +15,64 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Message create(Message entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
```

Comment:
let's keep our code consistent, use either `save()` or `update()` everywhere with corresponding cascade types where needed

Reply:
I didn't quite understand what was required of me. If i need to use CascadeType Persist i should use persist () method, if not then save ()? And same for remove() & delete()?

Reply:
no, you should use same method everywhere